meta-python/recipes-devtools/python/python3-posix-ipc_1.1.1.bb

```diff
@@ -10,3 +10,7 @@ UPSTREAM_CHECK_PYPI_PACKAGE = "${PYPI_PACKAGE}"
 SRC_URI[sha256sum] = "e2456ba0cfb2ee5ba14121450e8d825b3c4a1461fca0761220aab66d4111cbb7"
 
 inherit setuptools3 pypi
+
+# Ensure posix message queue support in python package for prober.py script
+DEPENDS += "glibc"
```

Comment:
glibc is default dependency added to every recipe if the distro you are building is using `TCLIBC = "glibc"` so I wonder if you are using TCLIBC = "musl" ?

Reply:
hmm, yes. it's not a solution